lib/ui/painting/path.h

```diff
@@ -28,6 +29,12 @@ class CanvasPath : public fxl::RefCountedThreadSafe<CanvasPath>,
     return fxl::MakeRefCounted<CanvasPath>();
   }
 
+  static fxl::RefPtr<CanvasPath> CreateFrom(SkPath src) {
+    fxl::RefPtr<CanvasPath> path = CanvasPath::Create();
+    path->path_ = src;
```

Comment:
`std::move(src)` or make the the argument `const SkPath& src` is you absolutely need the copy.